lib/simplecov/configuration.rb

```diff
@@ -301,5 +319,23 @@ def parse_filter(filter_argument = nil, &filter_proc)
         raise ArgumentError, "Please specify either a filter or a block to filter with"
       end
     end
+
+    #
+    # Check if use_branchable_report is called on a ruby version
+    # which not supporting it (< 2.5).
+    # Shows notify meessage and continue the process as normal
+    # lines coverage report on not supportable versions.
+    #
+    def target_capability(target)
+      if (RUBY_VERSION.to_f < 2.5) && target
```

Comment:
is there another way to check for this, maybe just `Coverage.method(:start).arity` ?

Reply:
Yes, using `arity` looks nicer.